go.mod

```diff
@@ -3,22 +3,58 @@ module github.com/datastax/cass-operator
 go 1.13
 
 require (
-	github.com/datastax/cass-operator/mage v0.0.0-00010101000000-000000000000
-	github.com/emicklei/go-restful-swagger12 v0.0.0-20170926063155-7524189396c6 // indirect
-	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b // indirect
-	github.com/gorilla/schema v1.1.0 // indirect
-	github.com/gorilla/websocket v1.4.2 // indirect
-	github.com/inconshreveable/go-vhost v0.0.0-20160627193104-06d84117953b // indirect
-	github.com/jessevdk/go-flags v1.4.0 // indirect
+	github.com/Jeffail/gabs v1.4.0
+	github.com/go-logr/logr v0.1.0
+	github.com/go-openapi/spec v0.19.0
+	github.com/gogo/protobuf v1.3.1 // indirect
+	github.com/google/uuid v1.1.1
 	github.com/magefile/mage v1.9.0
-	github.com/mjibson/appstats v0.0.0-20151004071057-0542d5f0e87e // indirect
 	github.com/onsi/ginkgo v1.11.0
 	github.com/onsi/gomega v1.8.1
-	github.com/rogpeppe/go-charset v0.0.0-20190617161244-0dc95cdf6f31 // indirect
+	github.com/operator-framework/operator-sdk v0.12.1-0.20191104165233-e1c79cae11cf
+	github.com/pkg/errors v0.8.1
+	github.com/spf13/pflag v1.0.5
+	github.com/stretchr/testify v1.4.0
+	golang.org/x/net v0.0.0-20191101175033-0deb6923b6d9 // indirect
+	golang.org/x/tools v0.0.0-20191219230827-5e752206af05 // indirect
 	google.golang.org/appengine v1.6.5 // indirect
-	gopkg.in/vmihailenco/msgpack.v2 v2.9.1 // indirect
+	gopkg.in/yaml.v2 v2.2.5
+	k8s.io/api v0.0.0
+	k8s.io/apimachinery v0.0.0
+	k8s.io/client-go v11.0.0+incompatible
+	k8s.io/code-generator v0.0.0
+	k8s.io/gengo v0.0.0-20191010091904-7fa3014cb28f
+	k8s.io/klog v1.0.0 // indirect
+	k8s.io/kube-openapi v0.0.0-20190918143330-0270cf2f1c1d
+	k8s.io/kubernetes v1.15.4
+	sigs.k8s.io/controller-runtime v0.3.0
+	sigs.k8s.io/controller-tools v0.2.2
 )
 
 replace github.com/datastax/cass-operator/mage => ./mage
```

Comment:
I think these replace directives are not needed anymore?

Reply:
good call